app/routes.py

```diff
@@ -61,9 +61,7 @@ def home():
 
 @app.route("/account")
 def user_account():
-    # user = get_user() 
-    return "Account Page"
-    #return render_template("account.html")
+    return render_template("account.html", title="Account")
```

Comment:
Why was this rolled back instead of building upon it? The account page template will eventually use the information returned from the `get_user()` function